moped-editor/src/views/staff/StaffListView.js

```diff
@@ -61,17 +63,17 @@ const staffColumns = [
   {
     headerName: "Workgroup",
     field: "moped_workgroup",
-    valueGetter: (props) => props.row.moped_workgroup.workgroup_name,
+    valueGetter: (value) => value.workgroup_name,
     width: 300,
   },
   {
     headerName: "Role",
     field: "roles",
-    renderCell: (props) => {
-      if (!props.value || !props.value[0]) {
+    valueGetter: value => {
```

Comment:
switching this from renderCell to valueGetter fixed the sort function for this column.